fms/Makefile.am

```diff
@@ -32,6 +32,9 @@ noinst_LTLIBRARIES = libfms.la
 # Each convenience library depends on its source.
 libfms_la_SOURCES = \
   fms.F90 \
+  include/fms.inc \
```

Comment:
are these also needed for fms_mod.$(FC_MODEXT)?

Reply:
850d957da3c0e28240215288b4de1aefdc05c132